prime-router/settings/staging/0167-baptist-alabama-staging-sender-receiver.yml

```diff
@@ -0,0 +1,112 @@
+# To submit this to localhost, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0167-baptist-alabama-staging-sender-receiver.yml
+#
+# Description:
+#   These following setting are for Baptist Health AL via Oracle RLN
+#   Order sender (baptist-alabama.etor-nbs-orders)
+#   Result Receiver (baptist-alabama.etor-nbs-results)
+#
+---
+- name: "baptist-alabama"
+  description: "Baptist Health AL via Oracle RLN"
+  jurisdiction: "STATE"
+  stateCode: AL
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders:
+    - !<full-elr>
+      name: "etor-nbs-orders"
+      organizationName: "baptist-alabama"
+      format: "HL7"
+      customerStatus: "active"
+      schemaName: ""
+      processingType: "async"
+      allowDuplicates: true
+      senderType: null
+      primarySubmissionMethod: null
+      topic: "etor-ti"
+      version: null
+      createdBy: null
+      createdAt: null
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "baptist-alabama"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/receivers/Flexion/TILabOrder"
```

Comment:
This schema is for orders. For results we don't have one you yet, but for now it would be `metadata/hl7_mapping/ORU_R01/ORU_R01-base`

Reply:
Thank you.